wouso/games/teamquest/models.py

```diff
@@ -2,19 +2,26 @@
 
 from wouso.core.game.models import Game
 from wouso.core.user.models import Player, PlayerGroup
+from wouso.core.qpool.models import Question
 
 
 class TeamQuestUser(Player):
-    group = models.ForeignKey('TeamQuestGroup', null=True, blank=True, default=None, related_name='users')
-	
-    def is_head(self):
-        if self.group is None:
-            return False
-        return self.group.head == self
+    pass
 
 
 class TeamQuest(models.Model):
-	pass
+    QUEST_TYPES = (
+        (TYPE_CLASSIC, 'In site text answers'),
+        (TYPE_CHECKER, 'In site answers, verified by checker'),
+        (TYPE_EXTERNAL, 'External levels and answers'),
+    )
+    start = models.DateTimeField()
+    end = models.DateTimeField()
+    title = models.CharField(default="", max_length=100)
+    questions = models.ManyToManyField(Question)
+    order = models.CharField(max_length=1000, default="", blank=True)
```

Comment:
Ok. Will make the changes

On Saturday, March 28, 2015, Iulian Gabriel Radu <notifications@github.com>
wrote:

> In wouso/games/teamquest/models.py
> <https://github.com/rosedu/wouso/pull/707#discussion_r27345187>:
>
> >
> >
> >  class TeamQuest(models.Model):
> > -	pass
> > +    QUEST_TYPES = (
> > +        (TYPE_CLASSIC, 'In site text answers'),
> > +        (TYPE_CHECKER, 'In site answers, verified by checker'),
> > +        (TYPE_EXTERNAL, 'External levels and answers'),
> > +    )
> > +    start = models.DateTimeField()
> > +    end = models.DateTimeField()
> > +    title = models.CharField(default="", max_length=100)
> > +    questions = models.ManyToManyField(Question)
> > +    order = models.CharField(max_length=1000, default="", blank=True)
>
> We are not going to have a sort feature for the team quest
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rosedu/wouso/pull/707/files#r27345187>.
>